classifai.php

```diff
@@ -186,3 +191,17 @@ function( $plugin_info, $http_response = null ) {
 		// @codingStandardsIgnoreEnd
 	}
 }
+
+/**
+ * Show dev version notice on ClassifAI pages if necessary.
+ */
+function classifai_dev_notice() {
+	if ( 0 === strpos( get_current_screen()->parent_base, 'classifai' ) ) {
+		?>
+		<div class="notice notice-warning">
+		<?php /* translators: %1$s: CLI install commands, %2$s: classifai url */ ?>
+		<p><?php echo wp_kses_post( sprintf( __( 'You appear to be running a development version of ClassifAI. Certain features may not work correctly without regularly running %1$s. If you&rsquo;re not sure what this means, you may want to <a href="%2$s">download and install</a> the stable version of ClassifAI instead.', 'distributor' ), '<code>composer install && npm install && npm run build</code>', 'https://classifaiplugin.com/' ) ); ?></p>
```

Comment:
This winds up outputting `Certain features may not work correctly without regularly running composer install && npm install && npm run build` which seems to imply you need to keep running composer install and npm install regularly. maybe just drop the word "regularly"?
```suggestion
		<p><?php echo wp_kses_post( sprintf( __( 'You appear to be running a development version of ClassifAI. Certain features may not work correctly without running %1$s. If you&rsquo;re not sure what this means, you may want to <a href="%2$s">download and install</a> the stable version of ClassifAI instead.', 'distributor' ), '<code>composer install && npm install && npm run build</code>', 'https://classifaiplugin.com/' ) ); ?></p>
```

Reply:
I suppose you don't need to run them regularly as in at a given interval or all that frequently but since this is really about people who are cloning from the GH repo and expecting things to work I would rather have the full initial build instructions. Dropping "regularly" seems fine, although hopefully it doesn't then imply you only have to do it once :)